### ISSUE TITLE
bug: React Router Redirect not working as expected

### ISSUE THREAD

**Issue by colinmac17** [ionic-team/ionic#20054]
# Bug Report
**Ionic version:**

[x] **4.11.5**

**Current behavior:**

I have a login component and my goal is to redirect to the home page once authenticated. However, when I login and set the App compoenent state to {authed: true} and redirect to home I stay on the login path. When I click the home tab I am able to get there but my goal would be to just redirect their automatically.

I see this error "No IonPage was found to render. Make sure you wrap your page with an IonPage", however I have IonPage wrapped around my Home component. I even tried wrapping IonPage around my redirect, but that did not help.

**Expected behavior:**

I expect that I will be redirected to the /home route and Home component.

**Steps to reproduce:**

**Related code:**
Login.js: https://gist.github.com/colinmac17/55c2d35af2a0b3106321fbd776669871
App.js: https://gist.github.com/colinmac17/d1e935cde3b0c4f1241fca30d91d8fa0

###Video

[react_router_bug.mov.zip](https://github.com/ionic-team/ionic/files/3944961/react_router_bug.mov.zip)

```
insert short code snippets here
```

**Other information:**

**Ionic info:**

```
insert the output from ionic info here
```

**Comment by elylucas** [comment]
Hi @colinmac17,

Would you mind testing with the latest dev build? I think we have some fixes in there already that will help with your issue:

npm i @ionic/react@4.11.5-dev.201912101612.e116712 @ionic/react-router@4.11.5-dev.201912101612.e116712

Thanks

**Comment by colinmac17** [comment]
@elylucas I got it to work using that build! When is that going to go live?

**Comment by elylucas** [comment]
@colinmac17 just published 4.11.6 which includes these changes. Thanks for testing!

**Comment by colinmac17** [comment]
@elylucas thanks! I will say that it was somewhat inconsistent and did not work every time. Side note, I noticed the link to the ionic worldwide slack group was not working http://ionicworldwide.herokuapp.com/

is there another way to join the channel?
